volmaster/daemon.go

```diff
@@ -71,25 +71,33 @@ func (d *DaemonConfig) Daemon(listen string) {
 	r := mux.NewRouter()
 
 	postRouter := map[string]func(http.ResponseWriter, *http.Request){
-		"/request":      d.handleRequest,
-		"/create":       d.handleCreate,
-		"/copy":         d.handleCopy,
-		"/mount":        d.handleMount,
-		"/mount-report": d.handleMountReport,
-		"/unmount":      d.handleUnmount,
-		"/remove":       d.handleRemove,
+		"/global":                           d.handleGlobalUpload,
+		"/volumes/create":                   d.handleCreate,
```

Comment:
Sure. So, I'm not too concerned about that right now because after 0.1 
we're going to gut this thing and replace it with something testable. :)

Let's make sure the code works and is tested, but for now at least, 
let's think/talk about what the future is like and maintain otherwise.

On 24 May 2016, at 9:33, Madhav Puri wrote:

>> @@ -71,25 +71,33 @@ func (d *DaemonConfig) Daemon(listen string) {
>>  	r := mux.NewRouter()
>>
>>  	postRouter := map[string]func(http.ResponseWriter, *http.Request){
>> -		"/request":      d.handleRequest,
>> -		"/create":       d.handleCreate,
>> -		"/copy":         d.handleCopy,
>> -		"/mount":        d.handleMount,
>> -		"/mount-report": d.handleMountReport,
>> -		"/unmount":      d.handleUnmount,
>> -		"/remove":       d.handleRemove,
>> +		"/global":                           d.handleGlobalUpload,
>> +		"/volumes/create":                   d.handleCreate,
>
> ah, I see, this should be fine as well.
>
> I was just wondering wrt consistency of the endpoints. Right now, in 
> policy and snapshot endpoints we are passing the resource name in the 
> URL but for volumes we don't, so thought I will check.
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/255/files/12004ac3ec0c06ee7937e32845ee33e7c45a120f#r64427347